android/build.gradle

```diff
@@ -34,6 +34,12 @@ repositories {
 dependencies {
     implementation 'com.braintreepayments.api:drop-in:4.+'
     implementation 'com.facebook.react:react-native:+'
+    // Braintree dependencies
+    implementation 'com.braintreepayments.api:braintree:3.7.0'
+    implementation 'com.braintreepayments.api:google-payment:3.2.0'
```

Comment:
After testing, you don't need to add this dependancy (it might be a sub deps of the drop-in)